<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>111</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="choose_dir_button">
      <property name="text">
       <string>Choose Dir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_dir_button">
      <property name="text">
       <string>Create Dir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dir_exists_button">
      <property name="text">
       <string>Dir Exist?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dir_or_file_button">
      <property name="text">
       <string>Dir or File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="folder_content_button">
      <property name="text">
       <string>Folder Content</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>218</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
